docs/tables/aws_lambda_function_event_source_mapping.md

```diff
@@ -0,0 +1,61 @@
+# Table: aws_lambda_function_event_source_mapping
+
+AWS Lambda is a compute service that lets you run code without provisioning or managing servers.
```

Comment:
```suggestion
```

I think it's ok if we just describe event source mappings in this table doc (else we'd be describing each service in each table doc)